core/tools/graph_compare.py

```diff
@@ -64,14 +70,18 @@ def k_most_similar(g, motif_db, k=5):
 
     return sorted(map(partial(compare_graphs, g), motif_db))[-k:]
 
-def k_most_similar_bp2(moduleLibraryPath, tempResponsePath):
+def k_most_similar_bp2(moduleLibraryPath, bp2Output, dataset):
     
-    with open(tempResponsePath) as f:
-        js_graph = json.load(f) #json output from BayesPairing2
+    js_graph = bp2Output
+
+    if (dataset.lower() == 'reliable'):
+        moduleLibraryPath += 'bayespairing_models_RELIABLE.json'
```

Comment:
Rename them to ALL.json and RELIABLE.json. GitLab adds the path when saving files (i..e it was stored in BayesPairing/models)

Reply:
lmao thats wild